specs/marketplace.md

```diff
@@ -185,178 +174,159 @@ struct Content {
 
 ```
 
-Notes about some parameters:
+The meaning of the Request parameters is given below:
+
+> Note that not all parameters are defined. I think that in a formal spec we need to introduce all of them, even shortly.
 
 `cid` 
 
 An identifier used to locate the Manifest representing the dataset.
-- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1) with sha-256 based [multihash](https://github.com/multiformats/multihash).
-- Data it represents SHOULD be discoverable in the network, otherwise the Request will get canceled. 
+- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1), SHA-256 [multihash](https://github.com/multiformats/multihash).
+- The data it represents SHOULD be discoverable in the network; otherwise, the Request MUST be canceled.
 
 `reward`
 
-- It MUST be an amount of tokens offered per slot per second.
-- The Ethereum address that submits the `requestStorage()` transaction MUST have [approval](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#IERC20-approve-address-uint256-) for transfer of at least the same amount on the ERC20 based token, that the network uses. 
+- MUST be an amount of Codex Tokens offered per slot per second.
```

Comment:
This is a tough one, because implementors following this spec will need to use the ERC-20 token used in the creation of the Marketplace contract (passed into the constructor). It's difficult to use a generic name when a specific, proper noun, is easier to comprehend imo, eg "CDX". An idea: in the terminology table, we could define "Token" (capitalised as it would be a proper noun), as "the address of an ERC-20 token used in the construction of the Marketplace contract.", and then use "Token" throughout the spec.

Reply:
Yeah, I really like the "Token" idea! 👍

Reply:
Super, I will use proper noun `Token` throughout the document.